client/components/Reports/SummarizeTestPlanReports.jsx

```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router-dom';
 import { Container, Table } from 'react-bootstrap';
 import styled from '@emotion/styled';
 import alphabetizeObjectBy from '../../utils/alphabetizeObjectBy';
-import getMetrics, { none } from './getMetrics';
```

Comment:
Yep they are very very similar! Minus React pieces having to be removed for the server side of course and slight differences in evaluating some filtered arrays.

But since it can't be shared across server and client when deployed, it has to be copied. But the calculations logic is the same and some parts still have to be used in the client. In the future, it might be worth removing the pieces from the respective server and client locations (which may come down to what needs to be returned for client vs server really) that aren't needed, but for now, I think it's fine.

I'll make a note on that future clean up!